_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title:  "Index Management UI enhancements"
+authors:
+- ihailong
+- gbinlong
+- suzhou
+- lxuesong
+date: 2023-02-28
+categories:
+ - technical-post
+meta_keywords: index management, index mappings, index operation UI, shrink index operations, OpenSearch Dashboards
+meta_description: Simplify cluster operations with Index Management UI enhancements that enable you to open, close, reindex, shrink, and split indexes in a more user-friendly way.
+
+excerpt: OpenSearch now provides a new interface you can use to execute common indexing and data stream operations. You can now perform create, read, update, delete (CRUD) and mapping operations for indexes, index templates, and aliases through the OpenSearch Dashboards UI. You can also open, close, reindex, shrink, and split indexes. This step marks the beginning of creating an Index Management UI, which will serve as a unified administration panel in OpenSearch Dashboards. 
+---
+
+Managing indexes, aliases, and templates in OpenSearch Dashboards can be difficult using only the API because there are restrictions and relation-binding tasks, such as adding aliases to indexes, simulating an index template by its name, and more. We are pleased to announce that these cluster operations have been significantly simplified with the Index Management UI enhancements in OpenSearch Dashboards v2.5.
+
+## Visual editor for index mappings with nested tree editor
+
+Building index mappings in pure JSON because it has several nested layers and properties. To simplify this task, we now provide a visual editor with editable nested properties, as shown in the following image. You can add properties by choosing the operation buttons and then see what the mappings will look like by switching to the JSON editor.
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/index-mapping-editor.png" alt="index mapping editor"/>{: .img-fluid }
+
+## Simulate an index by index name
+
+It is challenging to predict the appearance of the index when taking into account existing templates. To resolve this issue, we will attempt to match the index name with the templates whenever you change the index name, as shown in the image below. We will merge what users manually input with what the matching template contains, making it a "what you see is what you get" index.
```

Comment:
Instead of "we", do we mean "OpenSearch will attempt" and "OpenSearch will merge what users..."? "We" is unclear in these instances, so it's better to name what we're referring to.